tools/config/tsconfig-paths.ts

```diff
@@ -191,13 +212,66 @@ function handleSchematicsConfigs(
       }
     }
   });
+
+  if (showAllGood) {
+    success();
+  }
+}
+
+/**
```

Comment:
Why is setup library so special that it requires a separate config handler function?

Reply:
Answered offline